### DISCUSSION TITLE
Check log files in subfolders

### DISCUSSION THREAD

**Discussion by albertovincenzi** [opcodesio/log-viewer#15] [Ideas]
Current implementation doesn't allow to view logs in subfolders.

You might consider to implement showing folders tree and files in such folders as made here:
https://github.com/rap2hpoutre/laravel-log-viewer/blob/master/src/controllers/LogViewerController.php

**Comment by arukompas** [comment]
Great idea! I guess this also comes under the umbrella of "multiple log locations", where the logs aren't necessarily stored in the `storage/logs` folder.
